sealed-secrets.yaml

```diff
@@ -0,0 +1,118 @@
+package:
+  name: sealed-secrets
+  version: 0.28.0
+  epoch: 0
+  description: A Kubernetes controller and tool for one-way encrypted Secrets
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/bitnami-labs/sealed-secrets
+      tag: v${{package.version}}
+      expected-commit: 6b1b331a2cd3a58569ce4d819a7cabc59c0a3e50
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.36.0
+
+  - uses: go/build
+    with:
+      packages: ./cmd/controller
+      output: controller
+      ldflags: -X main.VERSION=${{package.version}}
+
+subpackages:
+  - name: ${{package.name}}-kubeseal
+    dependencies:
+      provides:
+        - kubeseal=${{package.full-version}}
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/kubeseal
+          output: kubeseal
+          ldflags: -X main.VERSION=${{package.version}}
+    test:
+      pipeline:
+        - runs: |
+            kubeseal --version | grep ${{package.version}}
+
+  - name: ${{package.name}}-kubeseal-bitnami-compat
+    description: "compat package with bitnami/sealed-secrets-kubeseal image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: sealed-secrets-kubeseal
+          version-path: 0/debian-12
```

Comment:
Let's use variables above for the path. `major-version` and `debian-version`

Reply:
We are not doing this in other places. Should we start doing this everywhere now?

It may be a good idea, if so, why not add it everywhere and require for new packages?

Reply:
Agreed on another approach for now